code/modules/projectiles/guns/mounted.dm

```diff
@@ -84,6 +84,209 @@
+	icon_state = "heavylaser"
+
+	fire_sound = 'sound/weapons/guns/fire/tank_flamethrower.ogg'
+	reload_sound = 'sound/weapons/guns/interact/minigun_cocked.ogg'
+
+	default_ammo_type = /obj/item/ammo_magazine/heavylaser
+
+	scatter = 10
+	deployed_scatter_change = -10
+	fire_delay = 0.7 SECONDS
+
+	flags_item = IS_DEPLOYABLE|TWOHANDED|DEPLOYED_NO_PICKUP|DEPLOY_ON_INITIALIZE
+	flags_gun_features = GUN_AMMO_COUNTER|GUN_DEPLOYED_FIRE_ONLY|GUN_WIELDED_FIRING_ONLY
+	gun_firemode_list = list(GUN_FIREMODE_AUTOMATIC)
+
+	attachable_allowed = list(
+		/obj/item/attachable/scope/unremovable/tl102/nest,
+	)
+
+	starting_attachment_types = list(
+		/obj/item/attachable/scope/unremovable/tl102/nest,
+	)
+
+	allowed_ammo_types = list(
+		/obj/item/ammo_magazine/heavylaser,
+	)
+
+	deploy_time = 5 SECONDS
+	undeploy_time = 3 SECONDS
+	deployed_item = /obj/machinery/deployable/mounted
+
+	max_integrity = 400
+	soft_armor = list("melee" = 0, "bullet" = 100, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 0, "fire" = 0, "acid" = 0)
+
+//-------------------------------------------------------
+//RR-15 mounted heavy recoilless rifle
+
+/obj/item/weapon/gun/heavyrr
+	name = "\improper RR-15 mounted heavy recoilless rifle"
+	desc = "The RR-15 mounted recoilless rifle is a non-IFF, modernized version of the L6 Wombat using 75mm. Reintroduced due to the rather close quarter nature of combat against xenomorphs, this thing will kill mostly anything on its way."
+
+	w_class = WEIGHT_CLASS_HUGE
+	flags_equip_slot = ITEM_SLOT_BACK
+	icon = 'icons/Marine/marine-hmg.dmi'
+	icon_state = "heavyrr"
+
+	fire_sound = 'sound/weapons/guns/fire/tank_cannon1.ogg'
+	reload_sound = 'sound/weapons/guns/interact/tat36_reload.ogg'
+	dry_fire_sound = 'sound/weapons/guns/fire/launcher_empty.ogg'
+
+	default_ammo_type = /obj/item/ammo_magazine/heavyrr
+	max_shells = 1 //codex
+	caliber = CALIBER_75MM // codex
+
+	scatter = 10
+	deployed_scatter_change = -10
+	windup_delay = 1 SECONDS
+	fire_delay = 3 SECONDS
+
+	flags_item = IS_DEPLOYABLE|TWOHANDED|DEPLOYED_NO_PICKUP|DEPLOY_ON_INITIALIZE
+	flags_gun_features = GUN_AMMO_COUNTER|GUN_DEPLOYED_FIRE_ONLY|GUN_WIELDED_FIRING_ONLY
+	gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO)
+
+	attachable_allowed = list(
+		/obj/item/attachable/scope/unremovable/tl102/nest,
+	)
+
+	starting_attachment_types = list(
+		/obj/item/attachable/scope/unremovable/tl102/nest,
+	)
+
+	allowed_ammo_types = list(
+		/obj/item/ammo_magazine/heavyrr,
+	)
+
+	deploy_time = 5 SECONDS
+	undeploy_time = 3 SECONDS
+	deployed_item = /obj/machinery/deployable/mounted
+
+	max_integrity = 600
+	soft_armor = list("melee" = 0, "bullet" = 100, "laser" = 0, "energy" = 0, "bomb" = 80, "bio" = 100, "rad" = 0, "fire" = 0, "acid" = 0)
+
+	///the smoke effect after firing
+	var/datum/effect_system/smoke_spread/smoke
+
+/obj/item/weapon/gun/heavyrr/Initialize(mapload, spawn_empty)
+	. = ..()
+	smoke = new(src, FALSE)
+
+/obj/item/weapon/gun/heavyrr/Destroy()
+	QDEL_NULL(smoke)
```

Comment:
why is this not a rr child so you dont have to copy paste this

Reply:
the backblast the rr creates damages the player behind it since this is a mounted gun the player has to be behind it so i have to make a new harmless smoke effect

Reply:
why not make it a bool whether it does damage